guidelines/terms/21/motion-animation.html

```diff
@@ -2,6 +2,6 @@
 <dd>
    					
    <p>addition of steps between conditions to create the illusion of movement or to give a sense of a smooth transition</p>
-	<p class="example">For example, an element which moves into place or changes size while appearing is considered to be animated. An element which appears instantly without transitioning is not using animation. Motion animation does not include changes of color, blurring or opacity.</p>
+	<p class="example">For example, an element which moves into place or changes size while appearing is considered to be animated. An element which appears instantly without transitioning is not using animation. Motion animation does not include changes of color, blurring or opacity which do not change the perceived size, shape, or position of the element.</p>
```

Comment:
per my additional concern in https://github.com/w3c/wcag/issues/1535#issuecomment-737906877 would it be worth also changing/expanding

> moves into place or changes size while ...

to

> moves into place or changes size or shape while ...

(as "size" may remain the same - in terms of area - but proportions/shape may morph)

Reply:
The first part is 'for example' so can include more. Also, the context is 'whilst appearing', so changing shape whilst appearing seems less common than changing size (e.g. expanding from an icon to full size).

Reply:
would just be nice to have it at least mentioned in passing (for things that do subtle stretchy animation things that do maintain size, but futz around with aspect ratio for instance), but yeh...not too fussed